src/LegacyStrings.jl

```diff
@@ -49,15 +49,15 @@ using Compat
     end
 
     if VERSION >= v"0.5.0-"
-        immutable ASCIIString <: DirectIndexString
+        struct ASCIIString <: DirectIndexString
```

Comment:
Does tagging a release prior to this pull request and then carrying on do the trick? Or does that approach present pitfalls?